www/source/partials/docs/_quickstart.html.md.erb

```diff
@@ -11,7 +11,7 @@ This guide covers how to quickly get started with Habitat
 
 Download the latest release of the Habitat cli tool <code>hab</code>. We are working on supporting tools like homebrew but for now simply visit our [downloads page](https://www.habitat.sh/download/).
 
-Looking for more information or if you'd like to grab the source of a specific release feel free to peruse our releases on github](https://github.com/habitat-sh/habitat/releases).
+Looking for more information or if you'd like to grab the source of a specific release feel free to peruse our releases on [github](https://github.com/habitat-sh/habitat/releases).
 
```

Comment:
i'm confused, i thought this was removing the quick start partial - where does this render?

Reply:
It is more a deactivation until it's ready.

Reply:
@eeyun can you confirm this is still a thing? :)